### PR TITLE
Spark example on PAI

### PR DESCRIPTION
**Pull request by hao1939**
Demo run Java/Python Spark application on PAI

### CODE REVIEW

**Comment by YanjieGao** [comment]
not seen jersey-client-1.9.jar and other jar replacement intro at README.md
Is it possible just give the download link and steps and avoid import these jars to pai repo?

**Comment by hao1939** [comment]
Good idea, I will try to avoid import these jars in PAI repo.
The doc is about how to use Spark on PAI.
What about add some comments in the Dockerfile?

**Comment by YanjieGao** [comment]
ok : )

File: examples/spark/README.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+```json
+{
+  "jobName": "spark-python-example",
+  "image": "openpai/spark-example",
+  "authFile": "",
+  "dataDir": "hdfs://YOUR_PAI_MASTER_IP:9000/user/core/data/mllib/",
+  "outputDir": "",
+  "codeDir": "hdfs://YOUR_PAI_MASTER_IP:9000/user/core/code",
+  "virtualCluster": "default",
+  "gpuType": "",
```
[the remark this answers is not in the document]
Comment (hao1939) [reply examples/spark/README.md]: Done

**Pull request merged**